docs/developer/develop-vscode.md

```diff
@@ -0,0 +1,135 @@
+
+#### Step 4 - Local Turbinia setup
+The local turbinia setup will use redis and celery. Let's create the configuration file for this setup.
+```
+$ sed -f ./docker/vscode/redis-config.sed ./turbinia/config/turbinia_config_tmpl.py > ~/.turbiniarc
+```
+
+#### Step 5 - Local Turbinia verification
+Let's verify that the local setup is working before we start developing and debugging. We are going to start a server and worker in seperate vscode terminals and create a Turbinia request in a third.. Open up 3 vscode terminals and execute below commands.
+
+Start server
+```
+$ python3 turbinia/turbiniactl.py -S server
+```
+Start worker
+```
+$ python3 turbinia/turbiniactl.py -S celeryworker
+```
+Fetch and process some evidence
+```
+$ curl https://raw.githubusercontent.com/obsidianforensics/hindsight/master/tests/fixtures/profiles/60/History > History
+$ tar -vzcf /evidence/history.tgz History
+$ python3 turbinia/turbiniactl.py compresseddirectory -l /evidence/history.tgz 
+$ python3 turbinia/turbiniactl.py status -r [request_id]
+```
+
+This should process the evidence and show output in each terminal for server and worker. Results will be stored in ```/evidence```.
+
+You're done :). Go to Step 8 for a vscode debugging example.
+
+#### Step 6 - GCP hybrid Turbinia setup
+Follow the ‘GCP Setup’ section [here](https://turbinia.readthedocs.io/en/latest/user/install-manual.html) and setup Cloud Functions, a GCE bucket, Datastore and PubSub.
+* Create a pubsub topic ,eg ‘turbinia-dev’.
+* Create a GCE storage bucket with a unique name.
+* Create the GCP Cloud Functions
+```
+$ git clone https://github.com/forseti-security/osdfir-infrastructure.git
+$ cd osdfir-infrastructure
+$ export TURBINIA_REGION=us-central1
+$ gcloud -q functions deploy gettasks --region $TURBINIA_REGION --source modules/turbinia/data/ --runtime nodejs10 --trigger-http --memory 256MB --timeout 60s
+$ gcloud -q functions deploy closetask --region $TURBINIA_REGION --source modules/turbinia/data/ --runtime nodejs10 --trigger-http --memory 256MB --timeout 60s
+$ gcloud -q functions deploy closetasks  --region $TURBINIA_REGION --source modules/turbinia/data/ --runtime nodejs10 --trigger-http --memory 256MB --timeout 60s
+```
+
+Create the Turbinia hybrid configuration file.
+```
+sed -f ./docker/vscode/psq-config.sed ./turbinia/config/turbinia_config_tmpl.py > ~/.turbiniarc
+```
+
+Edit the configuration file ```~/.turbiniarc``` and set below variables according to the GCP project you are using. Make sure all values are between quotes!
+```
+TURBINIA_PROJECT = '[your_gcp_project_name]' 
+TURBINIA_REGION = '[your_preferred_region]'  eg 'us-central1'
+TURBINIA_ZONE = '[your_preferred_zone]'  eg 'us-central1-f'
+PUBSUB_TOPIC = '[your_gcp_pubsub_topic_name]' eg 'turbinia-dev'
+BUCKET_NAME = '[your_gcp_bucket_name]'
+```
+
+Setup authentication for the GCP project.
+```
+$ gcloud auth login
+$ gcloud auth application-default login
+```
+
+#### Step 7 - GCP hybrid Turbinia verification
```

Comment:
I think step 5 and 7 are the same (maybe with the exception of the GCS bucket comment).  Maybe we combine them?